### ISSUE TITLE
Internal Placeholders to display own claims

### ISSUE THREAD

**Issue by RHX0R3** [WasabiThumb/xclaim#88]
Dear author and contributors,

I'd like to see a future update, where you implement custom placeholders.
Those would be great to implement and display the players own claims, so you can look at the current total claims and the maximum claims the player can set. An implementation example would be a scoreboard plugin like **[FeatherBoard](https://www.spigotmc.org/resources/featherboard.2691/)**.

### Example placeholders:

```
%xclaim_player_claims%
```
Total claims of the player.

```
%xclaim_player_claims_in_world%
```
Total claims of the player, in this particular world _W_.

```
%xclaim_player_claims_max%
```
Maximum claims the player can set in the current group.

I know, that this is not a top priority, as it is just a enhancement / feature request.
Also, you have any right to dismiss that idea.
But I'd like to see that in a future update, and I believe, that other users would like that too.

Thank you for reading!

_Best regards,
X0R3_

**Comment by WasabiThumb** [comment]
What you're asking for is [PlaceholderAPI integration](https://hangar.papermc.io/HelpChat/PlaceholderAPI), which is a reasonable request. After integrating with PlaceholderAPI, other plugins that use PlaceholderAPI (such as FeatherBoard) will be able to access placeholders from this plugin. This is similar to how Vault integration translates to essentially universal economy integration. I've done this before for other plugins and I understand that it should be pretty simple to add to this one. Will let you know (or you can try yourself on a fork if up to the task).

**Comment by WasabiThumb** [comment]
It has been [minimally implemented and staged for version 1.13](https://github.com/WasabiThumb/xclaim/commit/12d7b6cf4c3450185f2fe58ed34670afd7d33a5f). I've attached a development build as a courtesy, but I would recommend waiting until I've gotten a chance to test everything and make 1.13 public before putting it on a live server. [Here](https://github.com/WasabiThumb/xclaim/tree/dev/1.13?tab=readme-ov-file#placeholders) is a list of all the placeholders I've added so far. Feel free to suggest more or test yourself.

[1.13 Dev Build](https://send.zcyph.cc/download/b61906d609e2f41b/#Z3Puc40yQeRnE6PODuFq5g)

![XClaim_1_13_0_jar](https://github.com/user-attachments/assets/12d08284-4a00-4bdf-a154-eb214e4ec9ec)

**Comment by WasabiThumb** [comment]
I foolishly forgot that world names can and do contain underscores. Fixed that, seems to work perfectly now. This issue will be automatically closed when 1.13 is merged into master.

**Comment by WasabiThumb** [comment]
[1.13.0](https://github.com/WasabiThumb/xclaim/releases/tag/1.13.0) is here 🎊
